dm/worker/subtask_holder.go

```diff
@@ -46,6 +46,18 @@ func (h *subTaskHolder) removeSubTask(name string) {
 	delete(h.subTasks, name)
 }
 
+// resetAllSubTasks does Close, change cfg.UseRelay then Init the subtasks
+func (h *subTaskHolder) resetAllSubTasks(useRelay bool) {
+	h.mu.Lock()
+	defer h.mu.Unlock()
+	for _, st := range h.subTasks {
+		stage := st.Stage()
+		st.Close()
```

Comment:
What's the difference between this and `st.Pause()`?

Reply:
re-run init, so syncer's cfg can use new binlog type